runtime/cni_integ_test.go

```diff
@@ -209,8 +355,51 @@ func writeCNIConf(path string, networkName string, nameserver string) error {
       "dns": {"nameservers": ["%s"]}
     },
     {
-      "type": "tc-redirect-tap"
+      "type": "%s"
     }
   ]
-}`, networkName, nameserver)), 0644)
+}`, networkName, nameserver, chainedPluginName)), 0644)
+}
+
+func runCommand(t *testing.T, name string, args ...string) {
```

Comment:
Yeah, right now because the tests run in a container the process will just die when the container exits, but there's no harm in using `CommandContext`, will update with that.